cocos/2d/CCFastTMXTiledMap.cpp

```diff
@@ -217,20 +215,16 @@ TMXLayer * TMXTiledMap::getLayer(const std::string& layerName) const
 
 TMXObjectGroup * TMXTiledMap::getObjectGroup(const std::string& groupName) const
 {
-    CCASSERT(!groupName.empty(), "Invalid group name!");
-    if (groupName.empty()) {
-        return nullptr;
-    }
-
-    if (!_objectGroups.empty())
+    CCASSERT(groupName.size() > 0, "Invalid group name!");
```

Comment:
In fact, the code are write in cocos2d-x first. Then I apply the changes on the lite repo. So I have no intention to remove the logic. I will add it back.